index.html

```diff
@@ -5,21 +5,21 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<link rel="stylesheet" type="text/css" href="assets/css/paper.css">  
 	<link rel="icon" href="assets/images/color-circle.png">
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/paper.js/0.12.2/paper-full.min.js"></script>
 </head>
 <body style="background: linear-gradient(90deg, #0086FF, #0066FF);">
 
-<header>
-	Primitive Paint
+<header class="head">
+	Primitive Paint <i class="fa fa-paint-brush" aria-hidden="true"></i>
 </header>
 	<div class="wrapper">
-		<a class="links" href="simplePen.html">Simple Pen</a>
-		<a class="links" href="rectangle.html">Rectangle</a>
-		<a class="links" href="circle.html">Circle</a>
-		<a class="links" href="ellipse.html">Ellipse</a>
+		<a class="links" href="simplePen.html">Simple Pen <i class="fa fa-pencil" aria-hidden="true"></i> </a>
+		<a class="links" href="rectangle.html">Rectangle <i class="fa fa-pencil-square-o" aria-hidden="true"></i> </a>
+		<a class="links" href="circle.html">Circle <i class="fa fa-circle-thin" aria-hidden="true"></i> </a>
+		<a class="links" href="ellipse.html">Ellipse <i class="fa fa-circle-thin" aria-hidden="true"> </i></a>
 	</div>
 	<div class="footer">
-		<p>Made by <a href="https://github.com/ExpressHermes">Shivam Yadav</a></p>
+		<p>Designed and coded with <i class="fa fa-heartbeat" aria-hidden="true"></i> <a href="https://github.com/ExpressHermes">Shivam Yadav & Team</a></p>
```

Comment:
Change the footer to `Designed with coffee by <a href="https://github.com/ExpressHermes/Primitive-Paint" target="_blank"> Shivam Yadav & Team`